test/global-table.test.ts

```diff
@@ -0,0 +1,53 @@
+import { StackProps, aws_dynamodb } from 'aws-cdk-lib';
+import * as cdk from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import { GlobalTable } from '../src/global-table';
+
+test('global table region is matched to stack region, when unspecified', () => {
+  const region = 'us-east-1';
+  const app = new cdk.App();
+  const newstack = new cdk.Stack(app, 'id', {
+    env: { region: region },
+  });
+  new GlobalTable(newstack, 'hello', {
+    partitionKey:
+    {
+      name: 'id',
+      type: ddb.AttributeType.STRING,
+    },
+  },
+  );
```

Comment:
To match our typical indentation style. It's a little bit more compact:
```suggestion
  new GlobalTable(newstack, 'hello', {
    partitionKey: {
      name: 'id',
      type: ddb.AttributeType.STRING,
    },
  });
```